base/applications/rapps/loaddlg.cpp

```diff
@@ -714,12 +736,21 @@ DWORD WINAPI CDownloadManager::ThreadFunc(LPVOID param)
         if (!InternetCrackUrlW(InfoArray[iAppId].szUrl, urlLength + 1, ICU_DECODE | ICU_ESCAPE, &urlComponents))
             goto end;
 
+        dwContentLen = 0;
+
         if (urlComponents.nScheme == INTERNET_SCHEME_HTTP || urlComponents.nScheme == INTERNET_SCHEME_HTTPS)
             HttpQueryInfoW(hFile, HTTP_QUERY_CONTENT_LENGTH | HTTP_QUERY_FLAG_NUMBER, &dwContentLen, &dwStatus, 0);
 
         if (urlComponents.nScheme == INTERNET_SCHEME_FTP)
             dwContentLen = FtpGetFileSize(hFile, &dwStatus);
 
+        if (dwContentLen == 0)
+        {
+            HWND Item = GetDlgItem(hDlg, IDC_DOWNLOAD_PROGRESS);
```

Comment:
`Item` is already in the scope and has such instance. [See line 609](https://github.com/reactos/reactos/pull/123/commits/53094e92542eb4050e414747c585216f9961b6fd#diff-ebd6997347358f863dd270aac8e0880cR609).

Reply:
Indeed yeah, I was too hurry yesterday. :sweat_smile: